adapters/gumgum/gumgum.go

```diff
@@ -176,6 +176,14 @@ func preprocess(imp *openrtb2.Imp) (*openrtb_ext.ExtImpGumGum, error) {
 		}
 	}
 
+	if gumgumExt.Product != "" {
+		var err error
+		imp.Ext, err = json.Marshal(map[string]string{"product": gumgumExt.Product})
+		if err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
@gg-natalia  should add /update json framework tests for PR change

Reply:
Added one file, ty